File: engine/model/src/main/scala/net/server/channel/packet/cash/UseVegaSpellPacket.scala
package net.server.channel.packet.cash

class UseVegaSpellPacket(private var _position: Short, private var _itemId: Int, private var _firstCheck: Boolean, private var _equipSlot: Byte, private var _secondCheck: Boolean, private var _useSlot: Byte) extends AbstractUseCashItemPacket(_position, _itemId) {
  def firstCheck: Boolean = _firstCheck

  def equipSlot: Byte = _equipSlot

  def secondCheck: Boolean = _secondCheck

  def useSlot: Byte = _useSlot
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/use/UseVegaSpellPacket.java
package net.server.channel.packet.cash.use;

public class UseVegaSpellPacket extends AbstractUseCashItemPacket {
   private final Boolean firstCheck;

   private final Byte equipSlot;

   private final Boolean secondCheck;

   private final Byte useSlot;

   public UseVegaSpellPacket(Short position, Integer itemId, Boolean firstCheck, Byte equipSlot, Boolean secondCheck, Byte useSlot) {
      super(position, itemId);
      this.firstCheck = firstCheck;
      this.equipSlot = equipSlot;
      this.secondCheck = secondCheck;
      this.useSlot = useSlot;
   }

   public Boolean firstCheck() {
      return firstCheck;
   }

   public Byte equipSlot() {
      return equipSlot;
   }

   public Boolean secondCheck() {
      return secondCheck;
   }

   public Byte useSlot() {
      return useSlot;
   }
}
